src/main/java/org/cactoos/io/Joined.java

```diff
@@ -40,15 +40,25 @@
      */
     private final Iterable<Input> inputs;
 
+    /**
+     * Ctor.
+     * @param inputs Iterable of inputs
+     */
+    public Joined(final Iterable<Input> inputs) {
```

Comment:
@ilyakharlamov Please, don't use the same name to a parameter and attribute (I recommend change the parameter name, to something like `ipts`).